engine/util/src/main/scala/pl/touk/nussknacker/engine/util/functions/math.scala

```diff
@@ -0,0 +1,62 @@
+package pl.touk.nussknacker.engine.util.functions
+
+import pl.touk.nussknacker.engine.api.{Documentation, ParamName}
+
+object math {
+
+  @Documentation(description = "Returns the greater of two {@code double} values.")
+  def maxDouble(@ParamName("a") a: Double, @ParamName("b") b: Double): Double =
+    Math.max(a, b)
+
+  @Documentation(description = "Returns the greater of two {@code long} values.")
+  def maxLong(@ParamName("a") a: Long, @ParamName("b") b: Long): Long =
+    Math.max(a, b)
+
+  @Documentation(description = "Returns the greater of two {@code int} values.")
+  def maxInt(@ParamName("a") a: Int, @ParamName("b") b: Int): Int =
+    Math.max(a, b)
+
+  @Documentation(description = "Returns the greater of two {@code int} values.")
+  def maxFloat(@ParamName("a") a: Float, @ParamName("b") b: Float): Float =
+    Math.max(a, b)
+
+  @Documentation(description = "Returns the smaller of two {@code double} values.")
+  def minDouble(@ParamName("a") a: Double, @ParamName("b") b: Double): Double =
+    Math.min(a, b)
+
+  @Documentation(description = "Returns the smaller of two {@code long} values.")
+  def minLong(@ParamName("a") a: Long, @ParamName("b") b: Long): Long =
+    Math.min(a, b)
+
+  @Documentation(description = "Returns the smaller of two {@code int} values.")
+  def minInt(@ParamName("a") a: Int, @ParamName("b") b: Int): Int =
+    Math.min(a, b)
+
+  @Documentation(description = "Returns the smaller of two {@code int} values.")
+  def minFloat(@ParamName("a") a: Float, @ParamName("b") b: Float): Float =
+    Math.min(a, b)
+
+  @Documentation(description = "Returns the value of the first argument raised to the power of the second argument")
+  def pow(@ParamName("a") a: Double, @ParamName("b") b: Double): Double =
+    Math.pow(a, b)
+
+  @Documentation(description = "Returns the absolute value of a {@code int} value.")
+  def absInt(@ParamName("a") a: Long): Long =
```

Comment:
Let's make one function with Number as argument and deal with conversions in implementation?